Ginger/Ginger/Actions/ActionEditPages/ActExcelEditPage.xaml

```diff
@@ -48,7 +48,7 @@
 
             <StackPanel>
                 <Label Style="{StaticResource @InputFieldLabelStyle}">Select Row(s) Where:</Label>
-                <Actions:UCValueExpression x:Name="SelectRowsWhereTextBox" Margin="0,0,105,0" ToolTip="[Column Name]=Value to filter the rows by, like: Available='Yes' or Num=1 or [Team Name]='Ginger'."/>
+                <Actions:UCValueExpression x:Name="SelectRowsWhereTextBox" Margin="0,0,105,0" ToolTip="[Column Name]=Value to filter the rows by, like: Available='Yes' or Num=1 or [Team Name]='Ginger' or Account=XYZ."/>
```

Comment:
Add examples with space and special characters

Reply:
ok